Projects/DataSource/Sources/Repository/LocationRepository.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  LocationRepository.swift
+//  DataSource
+//
+//  Created by 이동현 on 11/9/25.
+//
+
+import CoreLocation
+import Domain
+
+final class LocationRepository: NSObject, LocationRepositoryProtocol {
+    private let networkService = NetworkService.shared
+    private let locationManager = CLLocationManager()
+    private var continuation: CheckedContinuation<LocationEntity?, Never>?
+    private var authContinuation: CheckedContinuation<CLAuthorizationStatus, Never>?
+
+    override init() {
+        super.init()
+        
+        locationManager.desiredAccuracy = kCLLocationAccuracyNearestTenMeters
+        locationManager.delegate = self
+    }
+
+    func getCoordinate() async -> LocationEntity? {
```

Comment:
```suggestion
    func fetchCoordinate() async -> LocationEntity? {
```


```suggestion
    func loadCoordinate() async -> LocationEntity? {
```


```suggestion
    func requestCoordinate() async -> LocationEntity? {
```


저희 `get / set` 네이밍을 지양하자 라는 컨벤션이 있어가주구 요러한 네이밍은 어떠신지요 !!!

Reply:
악 요즘 자바하다보니까 깜빡했어요 ㅜㅜ 수정하겠습니다